common/static/coffee/spec/discussion/view/discussion_thread_view_spec.coffee

```diff
@@ -102,40 +157,59 @@ describe "DiscussionThreadView", ->
                 course_settings: DiscussionSpecHelper.makeCourseSettings()
             )
 
+        describe "responses", ->
+            it "can post a first response", ->
+                renderWithTestResponses(@view, 0)
```

Comment:
I find this test difficult to read. In particular, the "is-hidden" check below is not scoped specifically to the initial post or the response, so it isn't clear which of the two is editable.

Some comments explaining the initial state, and more specific selectors that verify the menu state of both the initial post and the response, would improve the test.